src/main/java/cat/nyaa/HamsterEcoHelper/utils/database/tables/TempStorageRepo_v2.java

```diff
@@ -0,0 +1,35 @@
+package cat.nyaa.HamsterEcoHelper.utils.database.tables;
+
+import cat.nyaa.nyaacore.database.DataColumn;
+import cat.nyaa.nyaacore.database.DataTable;
+import cat.nyaa.nyaacore.database.PrimaryKey;
+import cat.nyaa.nyaacore.utils.ItemStackUtils;
+import org.bukkit.inventory.ItemStack;
+
+import java.util.List;
+import java.util.UUID;
+
+@DataTable("temporary_storage_v2")
+public class TempStorageRepo_v2 {
```

Comment:
We may use some more descriptive name like `TempNbtStorageRepo`

Reply:
Or use something like `NbtTempStorageRepo`, other than append `v2` should be better.